sqlglot/parser.py

```diff
@@ -1692,6 +1715,19 @@ def _parse_udf_kwarg(self):
 
         return self.expression(exp.UserDefinedFunctionKwarg, this=this, kind=kind)
 
+    def _parse_udf_body(self):
+        if not self._match(TokenType.ALIAS):
+            return None
+        expression = self._parse_string()
+        if not expression:
+            self._match(TokenType.L_PAREN)
```

Comment:
this seems off,

can you just do
return self._parse_expression(), it should handle selects and parens and strings automaticalyl for you

Reply:
There's three ways to specify UDF bodies:

```
CREATE FUNCTION x() AS 'SELECT 1'
CREATE FUNCTION x() AS SELECT 1
CREATE FUNCTION x() AS (SELECT 1)
```

In order to transpile I am trying to capture just the body, not the container.

Reply:
Test case: https://github.com/tobymao/sqlglot/pull/508/files#diff-48950cdaa69b6d8abaa3a1207fc43f4676296c3d36b2dd74e782b9b48c202511R289-R295

Reply:
Looks like _parse_statement() works. I'll use that.